_topic_map.yml

```diff
@@ -616,6 +616,9 @@ Topics:
   - Name: Sysctls
     File: sysctls
     Distros: openshift-origin,openshift-enterprise
+  - Name: Encrypting Data at Rest
```

Comment:
REST is the appropriate spelling, it's short for Representational state transfer